subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/KotlinPluginSmokeTest.groovy

```diff
@@ -48,7 +48,7 @@ class KotlinPluginSmokeTest extends AbstractSmokeTest {
             androidBuildToolsVersion: androidBuildToolsVersion)
 
         when:
-        def build = runner('clean', 'testDebugUnitTestCoverage').build()
+        def build = runner('clean', 'testDebugUnitTestCoverage').forwardOutput().build()
```

Comment:
Is this intentional?

Reply:
Yes. Not seeing the test output in the failure report made the test really hard to analyze.